webapp/seleniumTests/test1.html

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
yeah , that was one of my first tests and so the filename for it was not very meaningful and I changed the name in the selenium ide and didn't realize it didnt change this one also.